src/lib/dom.js

```diff
@@ -755,6 +755,7 @@ export function isInline(elm, includeCodeAsBlock) {
  *
  * @param {HTMLElement} from
  * @param {HTMLElement} to
+ * @deprecated since v3.1.0
  */
 export function copyCSS(from, to) {
```

Comment:
Should a warning be logged to the console if someone's code calls this?

Reply:
That's a great idea!